main.go

```diff
@@ -93,3 +88,44 @@ func main() {
 
 	os.Exit(0)
 }
+
+func generateCycloneDx(config config.Config, lResults *linker.Results) {
+	if config.ChequeConfig.UseIQ {
+		dx := cyclonedx.Default(logger.GetLogger())
+		sbom := dx.FromCoordinates(lResults.Coordinates)
+
+		iqOptions := iq.Options{
+			User:        config.IQConfig.Username,
+			Token:       config.IQConfig.Token,
+			Application: "cheque",
```

Comment:
This is an interesting one, also a good candidate for config. Right now you'd be hard coded to one application in IQ Server.

Reply:
that's coming in the next issue/PR that's already made heh